### PR TITLE
feat: winter 22 release updates and npm dependencies updates

### PR DESCRIPTION
**Pull request by msrivastav13**
### What issues does this PR fix or reference?

Winter22 release updates. 

- Updates version to 53.0
-  Metadata upgrades for winter 53.0 for flexipages
-  Updates npm dependencies

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,18 +27,18 @@
...
         "@salesforce/eslint-plugin-lightning": "^0.1.1",
         "@salesforce/sfdx-lwc-jest": "^0.14.0",
-        "eslint": "^7.28.0",
+        "eslint": "^7.32.0",
         "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-import": "^2.23.4",
-        "eslint-plugin-jest": "^24.3.6",
+        "eslint-plugin-import": "^2.24.2",
+        "eslint-plugin-jest": "^24.5.0",
         "husky": "^6.0.0",
```
Comment (pozil) [inline package.json]: Some dependencies are not up to date in this list.
For example, Husky should be `7.0.2`.

**Review by pozil — APPROVED** [review]
Thanks for the update @msrivastav13.
Can you check my comments regarding dependency versions and merge the changes?

File: package.json (hunk 1 above)
Comment (msrivastav13) [reply package.json]: Everything is updated. Thank you for your review.

**Comment by msrivastav13** [comment]
@pozil The CI is failing because the Scratch org creation fails. I guess one reason could be now we are out of that window where we can use `preview` although that's contrary to dates here.

https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_scratch_orgs_version_selection.htm

Another reason is may be our DevHub is updated to Winter 22 already!

**Comment by pozil** [comment]
Thanks for the update. You're right, for some reason the dev hub is already on Winter '22 so the scratch org creation the `preview` flag is failing.

I'll force merge this PR so that we can ignore this error and CI will run again on `prerelease/winter22`.

**Pull request merged**